subsys/bluetooth/audio/csip_set_coordinator.c

```diff
@@ -187,6 +187,9 @@ static struct bt_csip_set_coordinator_svc_inst *get_next_active_instance(void)
 	svc_inst =  lookup_instance_by_set_info(member, active.info);
 	if (svc_inst == NULL) {
 		LOG_DBG("Failed to lookup instance by set_info %p", active.info);
+	} else if (svc_inst->conn == NULL) {
+		LOG_DBG("Next service instance %p disconnected", svc_inst);
```

Comment:
in terms of commit message, it seems to happen when a device disconnects.

disconnected()  will reset the whole svc instance including set_info, I did not understand why svc_inst in line 187 is not NULL when device disconnects

Reply:
That is a very good point. Will mark this as DNM until I've investigated. Thanks